bazel/astore/BUILD.bazel

```diff
@@ -1,4 +1,41 @@
+load("//bazel/astore:defs.bzl", "astore_download")
+
 exports_files([
     "astore_upload_file.sh",
     "astore_upload_dir.sh",
 ])
+
+astore_download(
+  name = "download_a_big_file",
+  arch = "amd64",
+  download_src = "kernel/ubuntu/5.8.0-1003-oracle-amd64.tar.gz",
+)
+
+# This has to be a python test, not a shell test, because
+# of https://github.com/bazelbuild/bazel/issues/15043
```

Comment:
we commonly work around that issue by using a filegroup indirection. Eg, put the data dependencies in the filegroup, have sh_test depend on the filegroup, and it works.

Reply:
Thanks, I didn't know that.